test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1524,6 +1498,106 @@ static public void testCheckCastRefClassOnNull() throws Throwable {
 		checkCastRefClassOnNull.invoke();
 	}
 
+
+	// The three following tests can be used to verify that flattened value types in arrays are handled properly by the GC. In the
+	// current state, these tests should pass when flattened is disabled but failed when it is enabled. The JIT does not have adequate 
+	// support so until it is added, these tests can be run with the -Xint option enabled to disable to JIT compiler.
+	// /*
+	//  * Maintain a buffer of flattened arrays with long-aligned valuetypes while keeping a certain amount of classes alive at any 
+	//  * single time. This forces the GC to unload the classes.
+	//  */
+	// @Test(priority=4)
```

Comment:
set the priority to 5 so it runs after the tests that create classes have completed